templates/Makefile.build-harness

```diff
@@ -67,6 +67,7 @@ endef
 .PHONY : init
 ## Init build-harness
 init::
+	if ! make --version | grep -q "GNU Make 4.4"; then echo "GNU Make 4.4 is required. Please upgrade."; fi
```

Comment:
Have you considered adding some osx instructions on how to do this via homebrew?

```
brew install gmake
```

otherwise the user may not know how to upgrade `make`.

Reply:
makes sense, adding